### PR TITLE
Move contributor ladder from drafts to live

### PR DESCRIPTION
**Pull request by carolynvs**
* Move contributor ladder to main branch
* Rename to CONTRIBUTOR_LADDER.md
* Fix typos

### CODE REVIEW

File: contributor-ladder.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    * [Maintainer](#maintainer)
+* [Inactivity](#inactivity)
+* [Involuntary Removal](#involuntary-removal)
+* [Stepping Down/Emeritus Process](#stepping-downemeritus-process)
+* [Contact](#contact)
+
+
+## Contributor Ladder
+
+Hello! We are excited that you want to learn more about our project contributor ladder! This contributor ladder outlines the different contributor roles within the project, along with the responsibilites and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows.  Our project members are happy to help you advance along the contributor ladder.
```
Comment (geekygirldawn) [inline contributor-ladder.md:24]: Typo in responsibilities.

File: contributor-ladder.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    * Helping other users
+    * Submitting bug reports
+    * Commenting on issues
+    * Trying out new releases
+    * Attending community events
+    * [TODO: Other examples of non-contribution participation]
+
+
+### Contributor
+<!-- This role describes people who have just started contributing, or who contribute occasionally but don't participate in project governance or have defined responsibilities.  Usually projects define either this level or Community Participant, but not both.  If you don't definte this role, make sure to copy over its requirements to Organization Member -->
```
Comment (geekygirldawn) [inline contributor-ladder.md:49]: Typo in define

File: contributor-ladder.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+
+### Organization Member
+<!--This role is used by many projects where you have to be a regular contributor to have the right to vote in project elections, or to be able to operate project
+CI/CD automation.  If these rights do not apply to you, then it makes sense to combine this role with the Contributor role.  For most projects, Org Member is the \
+lowest level with clearly defined responsibilities and rights, because it is tied to permissions on the project's code repository. -->
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: An Organization Member is an established contributor who regularly participates in the project. Organization Members have privileges in both project repositories and elections, and as such are expected to act in the interests of the whole project.
+
+An Organization Member must meet the responsibilites and has the requirements of a Contributor, plus:
```
Comment (geekygirldawn) [inline contributor-ladder.md:80]: Typo in responsibilities

File: contributor-ladder.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+* Inactivity is measured by:
+    * Periods of no contributions for longer than [TODO: Number] months
+    * Periods of no communication for longer than [TODO: Number] months
+* Consequences of being inactive include:
+    * Involuntary removal or demotion
+    * Being asked to move to Emeritus status
+
+## Involuntary Removal or Demotion
+
+Involuntary removal/demotion of a contributor happens when responsibilites and requirements aren't being met. This may include repeated pattern of inactivity, extended period of inactivity, a period of failing to meet the requirements of your role, and/or a violation of the Code of Conduct. This process is important because it protects the community and its deliverables while also opens up opportunities for new contributors to step in.
```
Comment (geekygirldawn) [inline contributor-ladder.md:213]: Typo in responsibilities

File: contributor-ladder.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+CI/CD automation.  If these rights do not apply to you, then it makes sense to combine this role with the Contributor role.  For most projects, Org Member is the \
+lowest level with clearly defined responsibilities and rights, because it is tied to permissions on the project's code repository. -->
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: An Organization Member is an established contributor who regularly participates in the project. Organization Members have privileges in both project repositories and elections, and as such are expected to act in the interests of the whole project.
+
+An Organization Member must meet the responsibilites and has the requirements of a Contributor, plus:
+
+* Responsibilities include:
+    * Continues to contribute regularly, as demonstrated by having at least [TODO: Number] [TODO: Metric] a year, as demonstrated by [TODO: contributor metrics source]. <!-- Example: "as demonstrated by having at least 50 Github contributions per year, as shown by Devstats"-->
```
Comment (geekygirldawn) [inline contributor-ladder.md:83]: We should fix the capitalization of GitHub throughout.

File: contributor-ladder.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+* Requirements:
+    * Must have successful contribitons to the project, including at least one of the following:
```
Comment (geekygirldawn) [inline contributor-ladder.md:85]: Typo in contributions

File: contributor-ladder.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A Reviewer has responsibility for specific code, documentation, test, or other project areas. They are collectively responsible, with other Reviewers, for reviewing all changes to those areas and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.
+
+Reviewers are responsible for a "specific area." This can be a specific code directory, driver, chapter of the docs, test job, event, or other clearly-defined project component that is smaller than an entire repository or subproject. Most often it is one or a set of directories in one or more Git repositories. The "specific area" below refers to this area of responsibility.
+
+Reviewers have all the rights and responsibilities of an Organization Member, plus:
+
+* Responsibilities include:
+    * Following the reviewing guide
+    * Reviewing most Pull Requests against their specific areas of responsiblity
```
Comment (geekygirldawn) [inline contributor-ladder.md:123]: Typo in responsibility

**Review by geekygirldawn — CHANGES_REQUESTED** [review]
I was proposing a slightly stripped down version this contributor ladder for another project, and while I was using the template, I noticed a few typos, which I've included here in the review.

I also got a comment from someone who didn't believe that it was implementable using GitHub's permissions. I made some suggestions for how to do it, but I guess maybe it isn't obvious to others? I wonder if we need some comments or maybe a companion document that provides some guidance about how this maps to GitHub permissions or pointers to things like Prow? This shouldn't gate the template going out, but it might be something we should think about improving later.

**Comment by carolynvs** [comment]
I am incapable of spotting typos so I am very grateful @geekygirldawn that you are catching them! 👍

**Comment by carolynvs** [comment]
Yeah we do have a GitHub section on the site where we could add task oriented docs on things like bots and security/config. What do you think?

https://contribute.cncf.io/maintainers/github/

**Comment by jberkus** [comment]
@geekygirldawn "I also got a comment from someone who didn't believe that it was implementable using GitHub's permissions."

What does "it" refer to here?  Not following.

**Review by jberkus — APPROVED** [review]
Looks good, thanks for typo fixes!

**Comment by jberkus** [comment]
> Yeah we do have a GitHub section on the site where we could add task oriented docs on things like bots and security/config. What do you think?
> 
> https://contribute.cncf.io/maintainers/github/

yeah, I think that's the right place for GH-specific implementation details of stuff like "Org Member".

**Comment by carolynvs** [comment]
@geekygirldawn I've fixed the problems you identified, can you take another look?

File: CONTRIBUTOR_LADDER.md [hunk 8]
```diff
@@ -0,0 +1,226 @@
...
+* Additional privileges:
+    * Has GitHub or CI/CD rights to approve pull requests in specific directories
+    * Can recommend and review other contributors to become Reviewers
+    
+<!-- TODO: define how this works with your specific system.  For example:  "Is listed as Approver in the OWNERS file for certain directories. -->
+
+
+The process of becoming a Reviewer is:
+<!-- TODO: define your exact process here.  What's below is given as an example process for a project that uses Owners files and has defined teams for each project area -->
+1. The contributor is nominated by opening a PR against the appropriate repository, which adds the to the OWNERS file for one or more directories.
```
Comment (geekygirldawn) [inline CONTRIBUTOR_LADDER.md:143]: Looks like this is missing a word. Probably

... adds the GitHub id to the OWNERS file ...

**Review by geekygirldawn — APPROVED** [review]
I find one more little typo, but overall this looks really good!

**Comment by carolynvs** [comment]
I don't wan to keep pushing changes to the live drafts branch, so I've created a new PR to finish this.

**Pull request closed without merging**